tests/test_utils.py

```diff
@@ -765,3 +765,166 @@ def test_csv_latin_1_encoding(self):
         self.assertEqual(lookup_dict[4]['header 2'], 5)
         self.assertEqual(lookup_dict[4]['header 3'], 'foo')
         self.assertEqual(lookup_dict[1]['header 1'], 1)
+
+
+class ReadCSVToDataframeTests(unittest.TestCase):
+    """Tests for natcap.invest.utils.read_csv_to_dataframe."""
+
+    def setUp(self):
+        """Make temporary directory for workspace."""
+        self.workspace_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        """Delete workspace."""
+        shutil.rmtree(self.workspace_dir)
+
+    def test_read_csv_to_dataframe(self):
+        """Test the default behavior"""

```

Comment:
Could I suggest that the docstring here be prefixed with `Utils: `, and the same applied to all of the `test_*` methods in this fixture?  It makes for easier skimming in the test logs.